bonita/content.md

```diff
@@ -0,0 +1,276 @@
+# What is Bonita BPM?
+
+Bonita BPM is an open-source business process management and workflow suite created in 2001. It was started in France National Institute for Research in Computer Science, and then had incubated several years inside of the French computer science company Groupe Bull. Since 2009, the development of Bonita is supported by a company dedicated to this activity: Bonitasoft.
+
+> [wikipedia.org/wiki/Bonita_BPM](http://en.wikipedia.org/wiki/Bonita_BPM)
+
+![logo](https://github.com/bonitasoft/docker/blob/master/bonita/7.0.0/logo.png?raw=true)
+
+# How to use this image
+
+## Quick start
+
+	docker run --name bonita -d -p 8080:8080 bonita
+
+This will start a container running the [Tomcat Bundle](http://documentation.bonitasoft.com/tomcat-bundle-2) with Bonita BPM Engine + Bonita BPM Portal. With no environment variables specified, it's as like if you have launched the bundle on your host using startup.{sh|bat} (with security hardening on REST and HTTP APIs, cf Security part). Bonita BPM uses a H2 database here.
+
+You can access the Bonita BPM Portal on http://localhost:8080/bonita and login using the default credentials: install / install
+
+## Link Bonita BPM to a database
+
+### MySQL
+
+[Increase the packet size](http://documentation.bonitasoft.com/database-configuration-2#mysqlspec) which is set by default to 1M:
+
+	mkdir -p ~/Documents/Docker/Volumes/custom_mysql
+	echo "[mysqld]" > ~/Documents/Docker/Volumes/custom_mysql/bonita.cnf
+	echo "max_allowed_packet=16M" >> ~/Documents/Docker/Volumes/custom_mysql/bonita.cnf
+
+Mount that directory location as /etc/mysql/conf.d inside the MySQL container:
+
+	docker run --name mydbmysql -v ~/Documents/Docker/Volumes/custom_mysql/:/etc/mysql/conf.d -e MYSQL_ROOT_PASSWORD=mysecretpassword -d mysql:5.5
+
+See the [official MySQL documentation](https://registry.hub.docker.com/_/mysql/) for more details.
+
+Start your application container to link it to the MySQL container:
+
+	docker run --name bonita_mysql --link mydbmysql:mysql -d -p 8080:8080 bonita
+
+### PostgreSQL
+
+[Set max_prepared_transactions to 100](http://documentation.bonitasoft.com/database-configuration-business-data-1):
+
+	mkdir -p ~/Documents/Docker/Volumes/custom_postgres
+	echo '#!/bin/bash' > ~/Documents/Docker/Volumes/custom_postgres/bonita.sh
+	echo 'sed -i "s/^.*max_prepared_transactions\s*=\s*\(.*\)$/max_prepared_transactions = 100/" "$PGDATA"/postgresql.conf' >> ~/Documents/Docker/Volumes/custom_postgres/bonita.sh
+	chmod +x ~/Documents/Docker/Volumes/custom_postgres/bonita.sh
+
+Mount that directory location as /docker-entrypoint-initdb.d inside the PostgreSQL container:
+
+	docker run --name mydbpostgres -v ~/Documents/Docker/Volumes/custom_postgres/:/docker-entrypoint-initdb.d -e POSTGRES_PASSWORD=mysecretpassword -d postgres:9.3
```

Comment:
Should probably simplify this local directory in this and previous lines since `~/` may not expand properly.  I think just `mkdir custom_postgres` and then this run would be `-v "$PWD"/custom_postgres/:/docker-entrypoint-initdb.d`.  What do you think?

Could even skip the `mkdir` and just `-v "$PWD"/bonita.sh:/docker-entrypoint-initdb.d/bonita.sh`.